src/unix/lwt_engine.ml

```diff
@@ -197,6 +197,53 @@ end
 
 class libev_deprecated = libev ()
 
+class libuv () = object
+  inherit abstract
+
+  val loop = Luv.Loop.init () |> Result.get_ok
```

Comment:
Is it a deliberate choice not to use libuv's default loop?

Reply:
That is probably better yes